setup.py

```diff
@@ -6,5 +6,5 @@
       py_modules=['doct'],
       description='A read-only dottable dictionary',
       url='http://github.com/NSLS-II/doct',
-      requires=['six']
+      requires=['six', 'humanize']
```

Comment:
This is not the correct key.  One of our original `setup.py` files had this mistake and we have propagated it everywhere.  Should be `install_requires`